File: app/src/main/java/com/tutv/android/datasource/dto/ResourceViewedDTO.java
package com.tutv.android.datasource.dto;

public class ResourceViewedDTO {

    private boolean viewedByUser;

    public ResourceViewedDTO() {}

    public ResourceViewedDTO(boolean userViewed) {
        this.viewedByUser = userViewed;
    }

    public boolean isViewedByUser() {
        return viewedByUser;
    }

    public void setViewedByUser(boolean viewedByUser) {
        this.viewedByUser = viewedByUser;
    }

}

File: app/src/main/java/com/tutv/android/datasource/dto/ResourceViewedDTO.kt
package com.tutv.android.datasource.dto

import org.mockito.ArgumentMatchers
import org.mockito.InOrder
import org.mockito.Mockito

class ResourceViewedDTO {
    private var viewedByUser = false

    constructor() {}
    constructor(userViewed: Boolean) {
        viewedByUser = userViewed
    }

    fun isViewedByUser(): Boolean {
        return viewedByUser
    }

    fun setViewedByUser(viewedByUser: Boolean) {
        this.viewedByUser = viewedByUser
    }
}